modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -415,6 +451,19 @@ The following table lists the known issues and the version in which they were fi
 |===
 | Known issues | Fixed in
 
+| There is no Cypher syntax for creating a vector index.
+
+[TIP]
+====
+Use procedure to create the vector index.
```

Comment:
```suggestion
Use the procedure `db.index.vector.createNodeIndex` procedure to create the a vector index.
```

Reply:
```suggestion
Use the procedure `db.index.vector.createNodeIndex` to create the a vector index.
```
or 
```suggestion
Use the `db.index.vector.createNodeIndex` procedure to create the a vector index.
```
but not double procedure XD

Reply:
Sorry 😃 -  I prefer the first ("use the procedure...")

Reply:
Sorry 😃 - I prefer the first ("Use the procedure...")